### DISCUSSION TITLE
Package Update Frequency

### DISCUSSION THREAD

**Discussion by hilga007** [kth5/archpower#46] [General]
Hello - it looks like ISOs have not been updated for some time and packages are stale. 

Do you require additional compiling resources? 

I have a Ubiquiti set up at home and we could have you SSH to a dedicated box and set it up to automatically compile + publish packages 

OR you can guide me/train me through the process and I will gladly assist however I can / maintain as much as possible.

**Comment by kth5** [comment]
Packages aren't stale as per what build time is available. Currently that's 32 threads for ppc64le and ppc each as well as 8 HW threads for riscv64.

Ensure that you are using testing if you'd like to stay on the bleeding edge. At this time, a new snapshot of Archlinux's GIT mirror is taken, then all packages as per dependencies are being built/updated should they also have a package in Arch POWER.

Once that process is done with some manual testing and checking for further sobumps etc, [testing] is merged to [base]. This process can take quite some time.

**Comment by kth5** [comment]
The ISOs are working from what I can tell with personal testing. Since I haven't got the resources to keep up with archiso changes and they still work, it's on the todo list but well below updating packages.

**Reply by kth5** [reply]
New ISOs are uploaded and generated using a proper additon to a fork of archiso:
https://github.com/kth5/archiso

Release tag is imminent.

**Reply by iamhumanipromise** [reply]
Must say, this was AMAZING! I used Rufus from a Windows machine to DD write the iso to USB. 

I held "Option" on my PowerBook with the USB inserted. OpenFirmware found the GNU/GRUB loader AND ICON EVEN. Booted up just fine! 

PowerBook G4 1.0GHz w/512KB L2 cache
